src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Services/BulkUploadOrchestrationTests.cs

```diff
@@ -84,6 +84,24 @@ public async Task Should_Notify_Errors()
         _notificationService.Verify(ns => ns.SetErrorStatus(It.IsAny<string>(), It.IsAny<List<UploadFileErrorModel>>()), Times.Once());
     }
 
+    [TestMethod]
+    public async Task Should_Notify_Errors1()
```

Comment:
This should have a name describing when it notifies errors - the method above it should too

Reply:
changed